GTG/core/keyring.py

```diff
@@ -27,6 +33,31 @@
 from GTG.core.borg import Borg
 from GTG.core.logger import log
 
+class SecretKeyring(Borg):
+    def __init__(self):
+        super().__init__()
+        self._SECRET_SCHEMA = Secret.Schema.new("io.github.getting-things-gnome.gtg.v1",
```

Comment:
We use org.gnome.GTG everywhere (as is our app id). Is there any reason to use github here? (Also we will probably migrate to gitlab in the future)